changelog.md

```diff
@@ -5,6 +5,7 @@
 ### Refactor
 
 * [3332](https://github.com/zeta-chain/node/pull/3332) - implement orchestrator V2. Move BTC observer-signer to V2
+* [3368](https://github.com/zeta-chain/node/pull/3368) - refactor cli command to fetch inbound ballot from inbound hash and move to zetatools
```

Comment:
I would consider it as a feature instead, we really make this command usable for the user